### PR TITLE
sass initial configuration in project

### PR DESCRIPTION
**Pull request by unidrelyn**
## Summary 
the 'index' was created, the 'scss' folder is created, the compilation is done in 'sass' and it works correctly

## ScrennShots
screen shot 2018-09-19 at 4 15 12 pm

## TrelloCard
https://trello.com/c/lHzF7SZs/11-setup-del-proyecto

### CODE REVIEW

File: Maquetacion/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <h1 class="title">Hello World</h1> 
+</body>
+</html>
```
Comment (jmtov) [inline Maquetacion/index.html]: Remember to add a new line at the end of the files you're working on :)

File: Maquetacion/scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
+  }
```
Comment (jmtov) [inline Maquetacion/scss/application.scss]: Please make sure the opening and closing braces are at the same level.  Also remember the new line at the end of the file. 😬

**Review by jmtov — CHANGES_REQUESTED** [review]
Nice job! Please check out my comments. ✌️

**Review by jmtov — APPROVED** [review]
Nice job! 😸

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**